modules/cloud4/modules/integrations/pages/graphql.adoc

```diff
@@ -7,7 +7,7 @@ With TigerGraph GraphQL, you can use any GraphQL client or simply submit GraphQL
 
 [TIP]
 ====
-To learn more about GraphStudio read our xref:3.9@graphql:ROOT:index.adoc[].
+To learn more about GraphQL read our xref:3.9@graphql:ROOT:index.adoc[].
```

Comment:
```suggestion
To learn more about GraphQL read our xref:graphql:ROOT:index.adoc[].
```
@samzhou2 
Rules for when to include version or component name in the xref path:
1. If the link is an intra-component link (and if the component uses version numbers), DO NOT include the component name. If you do, it will jump to the latest version rather than staying with the version of the source page.
2. If you are crossing to another component, and the source and target use the same versioning scheme, e.g. server-docs to gsql-docs, use `@{page-component-version}` as the version, to stay on the same version.
3. If you are crossing to another component, and they use different version schemes, e.g. from cloud to graphql, there is no good way to get it right. Unless you want to permanently refer to a specific version, the best option is to omit the version number. You can add a note in the text letting the user know to consider the version.